test/certs/mkcert.sh

```diff
@@ -0,0 +1,417 @@
+#! /bin/bash
```

Comment:
Please mark in the file the part(s) adapted, or was an unchanged copy sufficient?